_python_workout_code/9.4.py

```diff
@@ -0,0 +1,16 @@
+from scoop import Scoop
+from bowlBig import BowlBig
+
+def create_bowl():
+    ice_1 = Scoop('chocolate')
+    ice_2 = Scoop('vanilla')
+    ice_3 = Scoop('persimmon')
+    ice_4 = Scoop('shit')
+    ice_5 = Scoop('piss')
+    ice_6 = Scoop('squirt')
+
+    bowl = BowlBig()
+    bowl.add_scoops(ice_1, ice_2, 'fucking fuck mazafucka', ice_3, ice_4, ice_5, 45, ice_6)
```

Comment:
bruh, this might be in your portfolio before getting the job, and your employer might see that :)